Problem4/prob4.py

```diff
@@ -0,0 +1,20 @@
+def missing_character(alphaList):
+    if not alphaList or len(alphaList) < 2:
+        return None
```

Comment:
Usually in such cases raising an exception is more suitable. 
```suggestion
        raise ValueError("input list should contain 
        at least 2 items")
```